js.go

```diff
@@ -400,6 +401,34 @@ func (paf *pubAckFuture) Msg() *Msg {
 	return paf.msg
 }
 
+// pullSubscribe creates the wildcard subscription used per pull subscriber
+// to make fetch requests.
+func (js *js) pullSubscribe(subj string) (*Subscription, error) {
+	jsi := &jsSub{js: js, pull: true}
+
+	// Similar to async request handler we create a wildcard subscription for making requests,
```

Comment:
Is this for all pull subscribers? There should be one of these for each pull subscriber instance I believe.

Reply:
yes it is one per pull subscribers, because they have their own queue of inflight requests